src/pages/ConvertVrt/Convert/index.tsx

```diff
@@ -0,0 +1,204 @@
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeHours',
+    };
+  } else if (vestingTimeRemainingMs < HOUR_IN_MILISECONDS) {
+    relativeTimeValues = {
+      realtiveTimeFormatValues: {
+        count: Math.floor(vestingTimeRemainingMs / MINUTE_IN_MILISECONDS),
+      },
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeMinutes',
+    };
+  } else if (vestingTimeRemainingMs < DAY_IN_MILISECONDS) {
+    relativeTimeValues = {
+      realtiveTimeFormatValues: {
+        hours: Math.floor(vestingTimeRemainingMs / HOUR_IN_MILISECONDS),
+        minutes: Math.floor(
+          ((vestingTimeRemainingMs % HOUR_IN_MILISECONDS) / HOUR_IN_MILISECONDS) * 60,
+        ),
+      },
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeHoursAndMinutes',
+    };
+  }
+  return relativeTimeValues;
+};
+
+const Convert: React.FC<ConvertProps> = ({
+  xvsTotalWei,
+  xvsToVrtRate,
+  vrtLimitUsedWei,
+  vrtLimitWei,
+  vrtConversionEndTime,
+}) => {
+  const styles = useStyles();
+  const { t, Trans } = useTranslation();
+  const readableXvsAvailable = useConvertToReadableCoinString({
+    valueWei: xvsTotalWei,
+    tokenId: XVS_ID,
+  });
+  const { relativeTimeTranslationKey, realtiveTimeFormatValues } =
+    formatI18nextRelativetimeValues(vrtConversionEndTime);
+
+  return (
+    <div css={styles.root}>
+      <section css={styles.title}>
+        <Typography variant="h3">{readableXvsAvailable}</Typography>
+        <Typography variant="small2">{t('convertVrt.xvsAVailable')}</Typography>
+      </section>
+      <ConnectWallet message={t('convertVrt.connectWalletToConvertVrtToXvs')}>
+        <AmountForm onSubmit={noop} maxAmount={vrtLimitWei.toFixed()}>
+          {({ values }) => {
+            const xvsValue = useMemo(() => {
+              if (values.amount) {
+                return new BigNumber(values.amount).times(xvsToVrtRate).toFixed();
+              }
+              return '';
+            }, [values.amount, xvsToVrtRate]);
+            return (
+              <>
+                <div css={styles.inputSection}>
+                  <Typography variant="small2" css={styles.inputLabel}>
+                    {t('convertVrt.convertVrt')}
+                  </Typography>
+                  <FormikTokenTextField
+                    tokenId={VRT_ID}
+                    name="amount"
+                    css={styles.input}
+                    description={
+                      <Trans
+                        i18nKey="convertVrt.balance"
+                        components={{
+                          White: <span css={styles.whiteLabel} />,
+                        }}
+                        values={{ amount: format(new BigNumber(vrtLimitWei)) }}
+                      />
+                    }
+                    rightMaxButton={{
+                      label: t('convertVrt.max').toUpperCase(),
+                      valueOnClick: xvsTotalWei.toFixed(),
```

Comment:
This needs to be converted to whole tokens. I also think it's the wrong value since this input is for VRT not XVS